src/bindings/python/BUILDING.md

```diff
@@ -1,248 +1,35 @@
-    -DENABLE_INTEL_GPU=OFF ^
-    -DENABLE_INTEL_MYRIAD_COMMON=OFF ^
-    -DENABLE_OV_ONNX_FRONTEND=ON ^
-    -DENABLE_PYTHON=ON ^
-    -DCMAKE_CXX_COMPILER="C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.27.29110\bin\Hostx64\x64"
-
-```
-
-There are a couple of things to notice here. One is that the full path to the x64 version of
-MSVC compiler has to be specified. This is because DNNL requires a 64-bit version and `cmake` may
-fail to detect it correctly.
-
-If you want to specify an exact Python version, use the following options:
-```
--DPYTHON_EXECUTABLE="C:\Program Files\Python37\python.exe" ^
--DPYTHON_LIBRARY="C:\Program Files\Python37\libs\python37.lib" ^
--DPYTHON_INCLUDE_DIR="C:\Program Files\Python37\include" ^
-```
-
-In order to build and install OpenVINO™, build the `install` target:
-
-    cmake --build . --target install --config Release -j 8
-
-In this step, OpenVINO™ is built and installed to the directory specified above. You can
-adjust the number of threads used in the building process to your machine's capabilities.
-
-Set up the OpenVINO™ environment in order to add a module path to `PYTHONPATH`:
-
-    %OPENVINO_BASEDIR%\openvino_dist\setupvars.bat
-
-### Build a Python Wheel on Windows
-
-Build the Python wheel package:
-
-    cd "%OPENVINO_BASEDIR%/openvino/src/bindings/python"
-    python setup.py bdist_wheel
-
-The final wheel should be located in the `bindings\python\dist` directory.
-
-    dir openvino/src/bindings/python/dist/
-    10/09/2020  04:06 PM         4,010,943 openvino-0.0.0.dev0-cp<version>-cp<version>-win_amd64.whl
-
-## Run Tests
-
-### Use a virtualenv (Optional)
+- ### Linux* Systems
+    Follow instructions available on [GitHub wiki pages for Linux build](https://github.com/openvinotoolkit/openvino/wiki/BuildingForLinux).
 
-You may wish to use a virutualenv for your installation.
+    Exact Python instructions can be found in [Additional Build Options section](https://github.com/openvinotoolkit/openvino/wiki/BuildingForLinux#additional-build-options).
 
-    $ virtualenv -p $(which python3) venv
-    $ source venv/bin/activate
-    (venv) $
+- ### Windows* Systems
+    Follow instructions available on [GitHub wiki pages for Windows build](https://github.com/openvinotoolkit/openvino/wiki/BuildingForWindows).
 
-### Install the Wheel and Other Requirements
+    Exact Python instructions can be found in [Additional Build Options section](https://github.com/openvinotoolkit/openvino/wiki/BuildingForWindows#additional-build-options).
 
-    (venv) $ cd "${OPENVINO_BASEDIR}/openvino/src/bindings/python"
-    (venv) $ pip3 install -r requirements.txt
-    (venv) $ pip3 install -r requirements_test.txt
-    (venv) $ pip3 install dist/openvino-0.0.0.dev0-cp38-cp38-linux_x86_64.whl
+- ### macOS* Systems for Intel CPU
+    Follow instructions available on [GitHub wiki pages for macOS build](https://github.com/openvinotoolkit/openvino/wiki/BuildingForMacOS_x86_64).
```

Comment:
I think we can use this instruction: https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md
And update it if it is needed.